src/helpers/BapiClient.ts

```diff
@@ -730,18 +726,9 @@ export class BapiClient {
       this.execute(createGetRedirectsEndpointRequest(params)),
 
     post: async (url: string) => {
-      try {
-        const response = await this.execute(
-          createPostRedirectEndpointRequest(url),
-        );
-        return response;
-      } catch (e) {
-        if (axios.isAxiosError(e) && e.response?.status === 404) {
-          return undefined;
-        }
-
-        throw e;
-      }
+      return await this.execute(
```

Comment:
TODO: Check this to return the correct error

Reply:
Mostly reverted so it should have the same behavior on 404 now